src/main/java/org/fundacionjala/coding/abel/CamelCase.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.abel;
+
+/**
+ *
+ */
+public class CamelCase {
+
+
+    /**
+     * @param str sting of imput
+     * @return value of return
+     */
+    public String camelCase(final String str) {
+
+        String[] string = str.split(" ");
+
+        for (int i = 0; i < string.length; i++) {
+            string[i] = convertUpercase(string[i]);
+        }
+        return String.join("", string);
+    }
+
+    /**
+     * @param str sting of imput
+     * @return value of return
+     */
+    private static String convertUpercase(final String str) {
+        if (str.isEmpty()) {
+            return str;
+        } else {
+            return String.valueOf(Character.toUpperCase(str.charAt(0))).concat(str.substring(1));
+
+        }
```

Comment:
This may be refactored like this.
```java
        if (str.isEmpty()) {
            return str;
        }
        return String.valueOf(Character.toUpperCase(str.charAt(0))).concat(str.substring(1));
```